src/encoded/audit/experiment.py

```diff
@@ -4424,18 +4439,71 @@ def check_experiment_atac_encode4_qc_standards(experiment, files_structure):
                 elif rescue >= 2 or self_consistency >= 2:
                     yield AuditFailure('borderline replicate concordance', detail, level='WARNING')
 
-            if 'reproducible_peaks' in metric:
-                rep_peaks = metric['reproducible_peaks']
-                detail = (
-                    f'According to ENCODE4 standards, overlap peaks files in ATAC-seq assays processed '
-                    f'by the uniform processing pipeline should have > 150k reproducible peaks. '
-                    f'100-150k reproducible peaks is acceptable, and < 100k is not compliant. '
-                    f'{audit_link(path_to_text(atac_peaks_file["@id"]),atac_peaks_file["@id"])} '
-                    f' file has {rep_peaks}.')
-                if rep_peaks <= 150000 and rep_peaks >= 100000:
-                    yield AuditFailure('moderate number of reproducible peaks', detail, level='WARNING')
-                if rep_peaks < 100000:
-                    yield AuditFailure('insufficient number of reproducible peaks', detail, level='NOT_COMPLIANT')
+    # The better reproducible peaks value in overlap or IDR thresholded peaks is used in audit
+    reproducible_peaks_files = overlap_peaks_files + audit_idr_thresholded_peaks_files
+    peaks_report = {}
+    for f in reproducible_peaks_files:
+        output_type = f['output_type']
+        if 'biological_replicates' in f:
+            reps = f.get('biological_replicates')
+        else: reps = 0
```

Comment:
I think you should take "reps = 0" on the next line

Reply:
I ended up removing the 'else' statement entirely; I didn't need to use the variable unless the 'if' statement was true